tests/compile_upp.sh

```diff
@@ -79,28 +79,21 @@ fi
 PATHTR=${PATHTR:-$( cd ${MYDIR}/.. && pwd )}
 source ${PATHTR}/tests/detect_machine.sh
 
+if [[ $MACHINE_ID == "hera_c" ]]; then
+  echo "CentOS will soon be decommissioned on Hera, please utilize Rocky 8 Nodes (hfe05-9 + hfe10-12)."
+  exit 1
+fi
```

Comment:
The `detect_machine.sh` does not return a `MACHINE_ID = hera_c`.  This block is not necessary.

Reply:
My bad. I see the `detect_machine.sh` is returning `hera_c` for some `hera` nodes.  That is not the case in the `ufs-weather-model`.

Reply:
Right apologies, I should have mentioned there were some slight modifications to make it clear to users still on hera_c that there is an ongoing transition to the Rocky 8 nodes.

Reply:
@FernandoAndrade-NOAA The hera_c.lua was removed from this PR. No need to keep hera_c detection.